scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -1160,14 +1196,15 @@ void BPF_STRUCT_OPS(layered_dispatch, s32 cpu, struct task_struct *prev)
 
 	/* consume preempting layers first */
 	bpf_for(idx, 0, nr_layers) {
-		layer_idx = iter_layer_cpu_ctx(cctx->layer_idx, idx);
 		if (disable_topology) {
-			if (MEMBER_VPTR(layers, [layer_idx].preempt) && scx_bpf_consume(layer_idx))
```

Comment:
cc @samuelnair this is reverting back to the previous logic of disabled topology.